src/app/pages/splash/containers/splash-page.component.ts

```diff
@@ -50,11 +55,13 @@ export class SplashPageComponent {
     this.configService.migrateToLatestVersion()
     return this.configService
       .fetchConfigState(false)
+      .catch(e => this.showConfigError(LocKeys.PROTOCOL_ERROR_DESC))
```

Comment:
Are we sure this will always be because of protocol pull failure? If not it may be worth making the error message a bit generic.